hcl/hcl.go

```diff
@@ -184,20 +184,20 @@ func BuildRunners[T Blocks](config T, tmpDir string, c *client.APIClient, since,
 			return nil, fmt.Errorf("hcl.BuildRunners product received unexpected nil client, product=%s", cfg.Name)
 		}
 
-		// Build product's HTTPers
+		// Build product's HTTPs
 		gets, err := mapProductGETs(cfg.GETs, redactions, c)
 		if err != nil {
 			return nil, err
 		}
 		runners = append(runners, gets...)
 
 		// Identical code between Product and Host, but cfg's type must be resolved via the switch to access the fields
-		// Build copiers
-		copiers, err := mapCopies(cfg.Copies, redactions, dest)
+		// Build Copy runners
+		copys, err := mapCopies(cfg.Copies, redactions, dest)
```

Comment:
Hmm... Should this be `copies`?

Reply:
Yeah, I went back and forth on this one. I can change it back.

Reply:
Yeah, personally, I think it reads a little better - especially, since on this same line, we also have `mapCopies` and `cfg.Copies` both spelled that way.

Reply:
Never ask an Austrian to fix the grammar... 🇦🇹